Sources/Core/CTypes.swift

```diff
@@ -0,0 +1,7 @@
+// Copyright © 2021 Zoe Carver <zoe@zoecarver.com>.
```

Comment:
Not sure what the standard protocol is here, should all files be under your name, or should I use my name here because I created this file?

Reply:
You should use your name, as you are the author of the file.